solr/core/src/java/org/apache/solr/cloud/Overseer.java

```diff
@@ -68,7 +68,7 @@
   public static final String QUEUE_OPERATION = "operation";
 
   // System properties are used in tests to make them run fast
-  public static final int STATE_UPDATE_DELAY = Integer.getInteger("solr.OverseerStateUpdateDelay", 2000);  // delay between cloud state updates
+  public static final int STATE_UPDATE_DELAY = ZkStateReader.STATE_UPDATE_DELAY;
```

Comment:
Moved so that I could access this setting in ZkStateReader, but left an alias here for locality.